documentation/docs/steps/gctsExecuteABAPUnitTests.md

```diff
@@ -0,0 +1,42 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have gCTS installed and configured on your ABAP systems.
+
+Learn more about the SAP git-enabled Central Transport Sytem (gCTS) [here](https://help.sap.com/viewer/4a368c163b08418890a406d413933ba7/201909.001/en-US/f319b168e87e42149e25e13c08d002b9.html). With gCTS, ABAP developments on ABAP servers can be maintained in Git repositories.
+
+## ${docGenParameters}
+
+## ${docGenConfiguration}
+
+## ${docJenkinsPluginDependencies}
+
+## Example
+
+Example configuration for the use in a Jenkinsfile.
+
+```groovy
+gctsExecuteABAPUnitTests(
+  script: this,
+  host: "https://abap.server.com:port",
```

Comment:
If the parameter is called host, I would expect to pass a host to it: "abap.server.com". Does it work this way?
But, as this may be easily confused, we also allowed "abap.server.com" as well as "https://abap.server.com" for our steps - however I'm not sure about the port...


I think we should aim to keep the steps at least similar.

Reply:
This was discussed in a previous PR of mine and this solution was accepted. Therefore, I will not be changing this with priority. (It's documented so it should be fine for the moment.)